package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@lemonpeach/sequelize-to-json-schema",
+  "version": "0.0.0",
+  "description": "",
+  "main": "lib/index.js",
+  "scripts": {
+    "clean": "shx rm -rf lib",
+    "build": "npm run clean && babel src -d lib",
+    "lint": "eslint src",
+    "test": "jest src"
```

Comment:
Can we add a lint step into this before jest?

Reply:
done